cmd/policy_publish.go

```diff
@@ -32,16 +33,36 @@ import (
 
 func newPolicyPublishCmd() *cobra.Command {
 	var cmd = &cobra.Command{
-		Use:   "publish <orgName>/<policyPackName>",
-		Args:  cmdutil.ExactArgs(1),
+		Use:   "publish [org-name]",
+		Args:  cmdutil.MaximumNArgs(1),
 		Short: "Publish resource policies to the Pulumi service",
-		Long:  "Publish resource policies to the Pulumi service",
+		Long: "Publish resource policies to the Pulumi service\n" +
```

Comment:
Should `Publish resource policies to the Pulumi service` be `Publish a Policy Pack to the Pulumi service`?